Lib/zoneinfo/_tzpath.py

```diff
@@ -111,14 +111,14 @@ def available_timezones():
         determine if a given file on the time zone search path is to open it
         and check for the "magic string" at the beginning.
     """
-    from importlib import resources
+    from importlib.resources import files
 
     valid_zones = set()
 
     # Start with loading from the tzdata package if it exists: this has a
     # pre-assembled list of zones that only requires opening one file.
     try:
-        with resources.open_text("tzdata", "zones") as f:
+        with files("tzdata").joinpath("zones").open() as f:
```

Comment:
```suggestion
        with resources.files("tzdata").joinpath("zones").open("rt") as f:
```

Should we be explicit about opening this as text?

Reply:
I'm not sure "rt" is valid for all implementations. I'm confident it is for pathlib.Path objects, but I suspect it's underspecified for Traversables (the protocol for resources). On inspection, I see it is specified to be ['r' or 'rb'](https://docs.python.org/3/library/importlib.html#importlib.abc.Traversable.open), so I'd use "r" or leave the default.